Library/QuickPulseSender.ts

```diff
@@ -84,6 +88,19 @@ class QuickPulseSender {
             additionalHeaders.forEach(header => options.headers[header.name] = header.value);
         }
 
+        if (this._authorizationHandler) {
+            try {
+                // Add bearer token
+                await this._authorizationHandler.addAuthorizationHeader(options);
+            }
+            catch (authError) {
```

Comment:
Will exceptions from the async methods dep in the stack like "_tokenRefresher.refresh(..)" bubbleup properly to report a meaningful error?  We may have to do some logging in those methods to ensure we dont lose critical diagnostic info.

Reply:
Exceptions must bubble up, are you worried about a specific path?, I prefer to have single place handling the errors but we can add more handling if specific error paths are identified and logging could be useful